Srigowri_Deogirikar/HTMLAssignment3.html

```diff
@@ -0,0 +1,54 @@
+  <!Doctype html>
+  <html lang="en">
+  <head>
+      <meta charset="UTF-8">
+      <meta http-equiv="X-UA-Compatible" content="IE-edge">
+      <meta name="viewport" content="width=device-width,initial-scale=1.0">
+      <title>Assignment 3</title>
+      
+  </head>
+  <body>
+      <h1>Ambersoft</h1>
+      <ol>
+  <li style="color:rgb(77, 44, 44)">MI-MEAN</li>
+    
+    <ol style="list-style-type:upper-roman;">
+      <li style="color:rgb(77, 44, 44)"> MI-MEAN-6</li>
+    
+      <ul>
+          <li style="color:rgb(77, 44, 44)">Shweta</li>
+          <li style="color:rgb(77, 44, 44)">Ajay</li>
+          <li style="color:rgb(77, 44, 44)">Trupti</li>
+          </ul>
+          <li style="color:rgb(77, 44, 44)">MI-MEAN-7</li>
+    <ul>
+      <li style="color:rgb(77, 44, 44)">Sagar</li>
+      <li style="color:rgb(77, 44, 44)">Umesh</li>
+    </ul> 
+    <li style="color:rgb(77, 44, 44)">MI-MEAN-8</li>   
+    <ul>
+      <li style="color:rgb(77, 44, 44)">Mihika</li>
+      <li style="color:rgb(77, 44, 44)">Pallavi</li>
+      <li style="color:rgb(77, 44, 44)">Reva</li>
+    </ul>
+    <li style="color:rgb(77, 44, 44)">MI-MEAN-9</li>
+  <ul>
+      <li style="color:rgb(77, 44, 44)"> Anil</li>
+      <li style="color:rgb(77, 44, 44)">Manoj</li>
+  </ul>
+  </ol>
+
+  <li style="color:rgb(62, 55, 146)">SYSTEM</li>
+  <ol>
+  <li style="color:rgb(62, 55, 146)">SYSTEM-1</li>
+  <ul>
+      <li style="color:rgb(62, 55, 146)">Anu</li>
+      <li style="color:rgb(62, 55, 146)">Dipti</li>
+  </ul>
+  <li style="color:rgb(62, 55, 146)">SYSTEM-2</li>
+    <ul>
+      <li style="color:rgb(62, 55, 146)">Triveni</li>  
+      <li style="color:rgb(62, 55, 146)">Kalyani</li>
+  </ul>  </ol> </ol></ol>
+  </body>
+  </html>
```

Comment:
create proper indentation for all html files

Reply:
ok sir